src/main/java/frc/robot/subsystems/indexer/Indexer.java

```diff
@@ -148,4 +148,12 @@ public Command indexerOutCommand() {
   public Command indexerStopCommand() {
     return runOnce(this::indexerStop);
   }
+
+  public Command indexerOnIntakeOnCommand() {
```

Comment:
I don't think you need these. Try to copy how the commands are already done from the RobotContainer as much as possible so you can make sure those flows work still (beam brake, etc)

// Intake Manual In
    controller.rightBumper().whileTrue(indexer.getDefault(pivot::inHandoffZone));

Which calls this:
 run(this::indexOffIntakeOn)
                        .until(() -> inputs.intakeBeamBreakState)
                        .andThen(run(this::indexOffIntakeOff))
                        .until(pivotHandoff)
                        .andThen(run(this::indexOnIntakeOn))))